spec/spec_helper.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'rails_helper'
```

Comment:
Question: I haven't seen this before - doesn't rails_helper usually require spec_helper? What motivated this? I noticed your test doesn't require rails_helper at the top - does this somehow make that not required?

Reply:
This was weird! Yes, rails_helper usually requires spec_helper. Usually, I have `--require spec_helper` in my `.rspec` file and all of the path loading works as expected. But for this application, it would not load anything from `app/lib` unless I added this require (found the suggestion on StackOverflow). Is this some bug in the latest rails perhaps?